src/components/EventCard/styles.scss

```diff
@@ -19,7 +17,7 @@
       border-radius: 5px;
       width: 100%;
       height: 100%;
-      object-fit: cover !important;
+      object-fit: contain !important;
```

Comment:
Why the `!important` is needed here?

Reply:
This happens without `!important`
![image](https://user-images.githubusercontent.com/66505715/176552761-a98d835c-3cfe-44ab-9432-d49227db038e.png)